domain/account/usecase/deposit.go

```diff
@@ -8,9 +8,9 @@ func (c AccountUc) Deposit(CPF string, amount float64) error {
 
 	account, err := c.r.FindOne(CPF)
 	if err != nil {
-		c.log.WithError(err).Errorln("the account does not exist")
+		c.log.WithError(err).Errorln("failed to recover account")
```

Comment:
I suppose you meant retrieve than recover, recover, as word, it's like regain after failure (well, at least I've learned like this)